File: app/src/main/java/com/tebet/mojual/view/qualitydetail/OrderDetailFragmentProvider.java
package com.tebet.mojual.view.qualitydetail;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class OrderDetailFragmentProvider {

    @ContributesAndroidInjector(modules = OrderDetailFragmentModule.class)
    abstract OrderDetailFragment provideQualityFragmentFactory();
}

File: app/src/main/java/com/tebet/mojual/view/qualitydetail/OrderDetailFragmentProvider.kt
package com.tebet.mojual.view.qualitydetail

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class OrderDetailFragmentProvider {

    @ContributesAndroidInjector(modules = [OrderDetailFragmentModule::class])
    internal abstract fun provideQualityFragmentFactory(): OrderDetailFragment
}
